mcs/class/corlib/Test/System.Threading/WaitHandleTest.cs

```diff
@@ -409,7 +409,7 @@ public void WaitOneWithAbandonedMutex ()
 					m.WaitOne ();
 				});
 				thread1.Start ();
-				thread1.Join (1000);
+				thread1.Join (Timeout.Infinite);
```

Comment:
This is not right, it should be Assert.IsTrue (thread1.Join (<reasonable timeout>));